app.js

```diff
@@ -0,0 +1,12 @@
+const menuBtn = document.querySelector('.nav-hamburger');
+const navlinks = document.querySelector('.nav-links');
+
+menuBtn.addEventListener('click', () => {
+  navlinks.classList.toggle('activin');
+  menuBtn.classList.toggle('active');
+});
+
+document.querySelectorAll('.links-mob').forEach((link) => link.addEventListener('click', () => {
+  navlinks.classList.remove('activin');
+  menuBtn.classList.remove('active');
+}));
```

Comment:
### To match the Figma design:
- [ ] Kindly add the blur effect for the background of the menu.
- [ ] Kindly get rid of the white spacing at the top of the menu.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/161630043-8e163f51-d184-465a-b20e-5e684ab11064.png) | ![image](https://user-images.githubusercontent.com/24703171/161545794-204de2da-a871-4195-8dc7-b4b064aa7adc.png) |

Reply:
hi @gemmen29 , thanks, but I don't see those margins when I use the models for responsiveness for different phones. 
could you tell me the measures, so I can see it like you? I have tried it in different phones and it looks good in all of them.  Moreover, could you give a hint of why did you select the app.js for the blur effect? i've been having issues with that.
let me show you. 
![image](https://user-images.githubusercontent.com/88522494/161631144-bb41a948-27bc-4375-b877-bed79009119d.png)
![image](https://user-images.githubusercontent.com/88522494/161631176-5a0cc843-6607-45e5-b01b-e402e057cc2c.png)
![image](https://user-images.githubusercontent.com/88522494/161631222-bdc1fe2c-d457-4b9e-a42d-09234b47c0e6.png)

Reply:
Yes, it looks nice on your device this is how it looks on my device in width `375px`.
![image](https://user-images.githubusercontent.com/24703171/161632132-5d0ead55-f67e-47dc-a80e-a401ac2d9dbd.png)

it worked fine for me when I changed the height of the modal from `110vh` to `130vh`
![image](https://user-images.githubusercontent.com/24703171/161632628-9ce39e27-549f-4ee9-ad2a-8fc59a17d77c.png)

Reply:
@gemmen29 hi! i have already fix the margins, but could you give a hint of how to do the blur effect please? I have tried all afternoon and no result.

Reply:
I beg you.

Reply:
Yes, you can use this property [backdrop-filter](https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter)

Reply:
@gemmen29 yes, I have already tried it but it's now working, could you please check it out? it's the only thing it's missing and i don't know what else is going on. Please.

Reply:
![image](https://user-images.githubusercontent.com/88522494/161643348-8f83650a-7d9c-4c21-ab90-ff2217154fab.png)
@gemmen29

Reply:
Yes, let me check

Reply:
Just add `backdrop-filter: blur(6px);` for the `ul` the takes class `nav-links` and it will work.

Reply:
I appreciate your help, but it's not working.  @gemmen29 
![image](https://user-images.githubusercontent.com/88522494/161645866-cef3cdce-9f5a-42ef-a337-d16ecbfd9ab4.png)

Reply:
This is how it looks on my PC
![image](https://user-images.githubusercontent.com/24703171/161646251-403b940a-cf25-42ee-996f-a536975f6adc.png)

Reply:
Which browser you are using?

Reply:
![image](https://user-images.githubusercontent.com/88522494/161648821-6188db53-a5dd-4f7c-823d-f7873267db56.png)
@gemmen29  I gonna take a pill because I am having a terrible headache, I don't know what else to do.

Reply:
![image](https://user-images.githubusercontent.com/88522494/161648989-dec8654d-ffb1-42ae-8090-e92013aae5d5.png)
@gemmen29 I am gonna take a pill because I don't know what is happening.

Reply:
I am having a terrible headache. @gemmen29

Reply:
IT WAS A PROBLEM OF FIREFOX!!

Reply:
Yes, it works fine in Chrome.